security/pkg/nodeagent/sds/monitoring.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// totalPushCounts records total number of SDS pushes since server starts serving.
+	totalPushCounts = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
Great to see this, however we are now using open census metrics API to gather metrics.
Please take a look at https://github.com/istio/istio/pull/14854  to see how that API is used.

Reply:
I will modify the PR to use open census. Thanks.